### ISSUE TITLE
Extensions: "more" action does not have a descriptive title

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52618]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : WIN10
**Additional Details:**
ScreenReaderNVDA[2018.2],JAWS[18.0.5038]MAS1.3.1,Voceover[MACOS High Sierra 10.13.5 Beta (17F70a)]
**Repro Steps:**
1. Launch VS Code when NVDA is ON.
2. Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Navigate to more button.
4. Observe what Screenreader reads.

**Actual:**
Screenreader reads as More button.It does not give extra information about the button what is inside the button.
Same issue observed in Jaws and Voiceover  also.
**Expected:**
Screenreader should give extra information about the button.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User unable to get the information about the button.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**

[[Accessibility]A11y_VSCode_ViewExtension_NVDA2.pptx](https://github.com/Microsoft/vscode/files/2123968/Accessibility.A11y_VSCode_ViewExtension_NVDA2.pptx)

**Comment by isidorn** [comment]
We should make the title of the more action more descriptive. Feels like low hanging fruit -> july

**Comment by JacksonKearl** [comment]
@isidorn that "More" text is the same "More" used throughout the application's secondary actions, would we create a bespoke label for all of them, or find some way to generate it?

**Comment by isidorn** [comment]
@JacksonKearl let's try to use the same label for all of them. Consistency.

**Comment by JacksonKearl** [comment]
Oh so just "More Actions" or something?

**Comment by isidorn** [comment]
More Actions works for me.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I verified the bug on below environment and the issue is no longer reproduced hence close this bug.
Version: 1.27.0-insider
Commit: 3f8088b5c5f1d44b5e1af4a8e67448abbe66470e
Date: 2018-08-27T07:36:47.732Z
Electron: 2.0.7
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by isidorn** [comment]
@AccessibilityTestingTeam-TCS the issue is already closed, but thanks
